### PR TITLE
Runtime services proxy for Scalar

### PR DESCRIPTION
**Pull request by marcopiraccini**
This exports the API to start the runtime proxy (the start returns the URL of the proxy) and to stop, under the assumption that it's open one proxy at a given time (i.e. no multiple proxies started in parallel are supported).

### CODE REVIEW

File: src/main/lib/runtime-proxy.mjs [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+  async start (id, serviceId) {
+    logger.info({ id }, 'Starting proxy')
+    const pid = this.#applications.getPid(id)
+    if (!pid) throw new Error('Application running PID not found')
+
+    const socketPath = this.#getSocketPath(pid)
+
+    if (!this.#runtimeProxy) {
+      this.#runtimeProxy = Fastify({ logger: true, level: 'debug' })
```
Comment (mcollina) [inline src/main/lib/runtime-proxy.mjs:37]: You'll have problems with this on Windows when there is no stdout. Disable logging if it's not a final build.

**Review by mcollina — CHANGES_REQUESTED** [review]

File: src/main/lib/runtime-proxy.mjs (hunk 1 above)
Comment (marcopiraccini) [reply src/main/lib/runtime-proxy.mjs]: PTAL @mcollina

**Review by tonysnowboardunderthebridge — APPROVED** [review]
Lgtm

**Review by mcollina — APPROVED** [review]
lgtm

**Pull request merged**